db_operations.py

```diff
@@ -80,6 +80,15 @@ def create_listing(listing):
         return False
 
 
+def create_empty_listing():
+    try:
+        doc_ref = listings_db.document()
+        return doc_ref.id
+    except Exception as e:
+        log.error(e)
+        return False
```

Comment:
do you want to return None here? to standardise with the other functions

Reply:
Changed the False to None instead. But I would still need to the doc_ref.id as it automatically generates a document id for me